src/Adapter/Cart/QueryHandler/GetCartForOrderCreationHandler.php

```diff
@@ -107,15 +107,15 @@ public function __construct(
     }
 
     /**
-     * @param GetCartInformation $query
+     * @param GetCartForOrderCreation $query
      *
      * @return CartInformation
      *
      * @throws CartNotFoundException
      * @throws LocalizationException
      * @throws PrestaShopException
      */
-    public function handle(GetCartInformation $query): CartInformation
+    public function handle(GetCartForOrderCreation $query): CartInformation
```

Comment:
```suggestion
    public function handle(GetCartForOrderCreation $query): CartForOrderCreation
```

I think the query result class should be renamed as well for consistency